echopype/tests/metrics/test_metrics_summary_statistics.py

```diff
@@ -24,7 +24,7 @@ def create_test_ds(Sv, range):
     testDS = xr.Dataset(
         data_vars=dict(
             Sv=(["frequency", "ping_time", "range_bin"], Sv),
-            range=(["frequency", "ping_time", "range_bin"], range),
+            echo_range=(["frequency", "ping_time", "range_bin"], echo_range),
```

Comment:
I think the `range_bin` here should become `range_sample` (is there an "s" at the end? sorry I forgot...), and then corresponding changes in the test functions when in involves operations across that dimension.

Reply:
Oops I just realized what you said about how the `range_bin` to `range_sample` will be in another PR. Please ignore my comment above!!